composer.json

```diff
@@ -30,7 +30,8 @@
     "require-dev": {
         "phpunit/phpunit":         "^4.0|^5.1",
         "mockery/mockery":         "^0.9",
-        "cache/integration-tests": "^0.9"
+        "cache/integration-tests": "^0.9",
+        "symfony/cache":           "dev-master"
```

Comment:
Do we have a non-taggable PSR-6 cache that doesn't require a server to be up for testing?